### PR TITLE
Add configuration for host name in zabbix

### PR DESCRIPTION
**Pull request by teriyakichild**
Includes commits from #20

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -240,11 +240,14 @@ This controls where your Zabbix metrics are sent for collection.
...
 >
-> **`host`** is your Zabbix host:port. If you leave out a : port designator the default 10051 will be assumed.
+> **`host`** is the name of the host in zabbix used to store metrics.
+>
+> **`server`** is your Zabbix host:port. If you leave out a : port designator the default 10051 will be assumed.
 
     config:
       zabbix:
-        host: localhost:10051
+        host: api.net
```
Comment (gregswift) [inline README.md]: isn't this a major API breakage?

Comment (teriyakichild) [inline README.md]: @gregswift good point.

Comment (teriyakichild) [inline README.md]: @gregswift version bumped

File: url_monitor.spec [hunk 2]
```diff
@@ -53,31 +53,8 @@ mkdir -p %{buildroot}%{_localstatedir}/lib/%{name}
 %attr(0755,-,-) %{_bindir}/%{name}
 
 %changelog
```
Comment (gregswift) [inline url_monitor.spec]: why is the changelog removed and no new changelog entry for 2.0.0 ?

**Pull request merged**